### ISSUE TITLE
Sent email

### ISSUE THREAD

**Issue by damdinsharav** [mailcow/mailcow-dockerized#2207]
Hello,

Is there way to see all sent email log.

I know there is an in-memory log but it's not enough as it is dating back only a day or two. I need at least last month's email output.

The reason is I blocked by Yahoo and they telling me my server sent hundreds of email at once. 

Thanks in advance.

**Comment by mkuron** [comment]
Duplicate of #2178
